apps/criticalchicken/critical_chicken.star

```diff
@@ -0,0 +1,369 @@
+""")
+
+# First impressions
+imgFirst = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAlElEQVQ4jdWUsQqAMAxEU3Fztb9qv8FvdbW7Q+Aac6FVnAxSjubykqCYzm2VP8ckIst+6KMaNzDZLKf4hIF1JwWCE+EktwVEpJZcS7aJWjKgyNrTheWilrXWWqbVQIUEdrYF3Fpwuwk6wROEur/hsAU72xvouJ/3YJrjfOSzcx7WaEH42bwNoCyT+WrgjuEk6e9/oQtj2aH3JX/ovQAAAABJRU5ErkJggg==
+""")
+
+# Hands on
+imgHands = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAZElEQVQ4jWP8UibCMJQBEwMDA3fnaziJxsDKxuTi0kUHwITG5+58/bVcFO4CrGyImq/losToojVggrsA2TX49dDTfQQB1APIwYnMxgUGjx9YiFcKSScMsChC8zNcls6AcaiXQgDLbEH/hv0EJAAAAABJRU5ErkJggg==
+""")
+
+# Preview
+imgPreview = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAaUlEQVQ4jWP8UibCMJQBEwMDA3fna+7O1xA+JhvCxUpilR0ADzAwMHwtF4Vbj8b+Wi6KSzN+WfoAqAe4O1/DnYLGxqN5QIIcDSBiAC6EFgPIgmjhjcwdKM8w4ZdGzhL4ZQcqLTEO9VIIAECZQ3zUBQYwAAAAAElFTkSuQmCC
+""")
+
+# Second look
+imgSecond = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAf0lEQVQ4jdVUsQ3AIAwD1I21vFpu4FbWsncwtVCCkFpY8IJjhTgBhL2v0+wMh8Wn7FMmAUfI1Fbscr1lUEFn/h/Ap1xiKDFAIodOM3LkdLnAoMIqON2B8OBga4F55uvUJyROVDjN22isuo3DvC2yb4bwgN5yAa1/rTADu/sv9ACud2p++XPSeAAAAABJRU5ErkJggg==
+""")
+
+# PRIORITY 1 TITLETAGS - PARENT CATEGORIES
+
+# News
+imgNews = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAYElEQVQ4jWO8xMTEMJQBEwMDg+7fv7p//0L4EDaEi0yiKRs8gAVCXWZm1v379zIzM4SNqQ4uO9gAlvSDNZghPqS9e0gGWDyAHNLIjh6cfsAeA3CHwj0DERyEqYhxqJdCAPtWMkGheN8eAAAAAElFTkSuQmCC
+""")
+
+# Feature
+imgFeature = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAbUlEQVQ4jWMUTbnDMJQBCwMDw6vZyhCOWOpdZDZECs5AVoNMIitG1ksfwAS3Em4rnI3pRKwuQ/M2PV3PAPcA3AV42LgAPJaIVE9dgIgBuBAuNi6AmfDoCZgwhV7NVsYfkPAMQIZeqgPGoV4KAQC/IEoZwsoIUAAAAABJRU5ErkJggg==
+""")
+
+# Review
+imgReview = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAZUlEQVQ4jWP8UibCMJQBEwMDA3fna+7O1xA+hA3hYiWxyg4gYIFQX8tFuTtffy0XhbDxaMAvS3/ABKHgrmcgFKgDHuRoAOoBSAzA2ciCaEGOzB0MnmHCFELOEvhlB0NyYhzqpRAAjPg3fsFfT3QAAAAASUVORK5CYII=
+""")
+
+# FALLBACK - IF NO RELEVANT CATEGORY IS FOUND
+
+# Our latest post
+imgLatest = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAeElEQVQ4jd1Syw6AMAgrxv//ZTwQCQHWoSdjDwuPUmgyAFBVAB746/XYraneSHEUSfWq37bqVIN0dPu2O6pzwl+lfGNNKw7uTURIl9dFhK9v9bdTCcwA11p5i/XXCnMPp7FNK8YtTJcQImdCjhcbcz71Ezz7cB/EBe+wuYw/ht85AAAAAElFTkSuQmCC
+""")
+
+# ERROR MESSAGE
+
+# Sorry...
+imgSorry = base64.decode("""
+iVBORw0KGgoAAAANSUhEUgAAAEAAAAAHCAIAAAA3VtxdAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAZklEQVQ4jdVUOQ7AIAwLeTHmIeTLHVxZEaIdOkDxgmNZwglHQa12MpxLj+gR4lNxMKjcBQZwsgY0QFzhnvhGaHwM7PaaTL7M6Ze+GMO+LnXaw29PQLiv0Ofp5vewpr2c08zK6b/QBeMeUr9B6cGnAAAAAElFTkSuQmCC
+""")
+
+# MAIN
+
+def main():
+    headline_cached = cache.get("cached_headline")
```

Comment:
Check out the built in cache functionality of the http module. Just pass`ttl_seconds` to `http.get()` and you get caching for free. Easier than using the cache module, and also more future proof as we may change the behavior and scope of the cache module in the future.